scripts/dev-bundle-tool-package.js

```diff
@@ -14,7 +14,7 @@ var packageJson = {
     npm: "4.6.1",
     "node-gyp": "3.6.0",
     "node-pre-gyp": "0.6.34",
-    "meteor-babel": "0.21.4",
+    "meteor-babel": "0.21.5",
```

Comment:
This change technically requires a new dev bundle, but I think we can wait on that because we don't minify the command-line tool implementation.